am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -132,27 +138,24 @@ void whenNoExplicitAccessShouldUseRoleBasedAccess() {
 
     @Test
     void whenNoExplicitAccessAndRoleHasExplicitAccessTypeShouldReturnNull() {
-        importerService.addRole(OTHER_ROLE_NAME, RoleType.RESOURCE, PUBLIC, EXPLICIT);
-
-        FilteredResourceEnvelope result = service.filterResource(accessorId, ImmutableSet.of(OTHER_ROLE_NAME),
+        FilteredResourceEnvelope result = service.filterResource(accessorId, ImmutableSet.of(otherRoleName),
```

Comment:
I'd say that this makes the test less readable. No pre and out of sudden there is call to filter method. To keep setup separate from method I'd suggest renaming `otherRoleName` to be `idamRoleWithExplicitAccess`. The same way `roleName` would become `idamRoleWithRoleBaseAccess`.

Reply:
👍